package.json

```diff
@@ -37,7 +37,8 @@
     "chalk": "^2.0.1",
     "columnify": "^1.5.4",
     "commander": "^2.11.0",
-    "cosmiconfig": "davidtheclark/cosmiconfig#3.0",
+    "cosmiconfig":
+      "davidtheclark/cosmiconfig#6eb3cc85d3a435e6b9e252b341e601960312f410",
```

Comment:
Is this because the `3.0` branch was merged into `master` just not released yet? I'm asking because the commit `6eb3cc85d3a435e6b9e252b341e601960312f410` is currently `HEAD` of `master`.

Reply:
Yeah they merged and then deleted the 3.0 branch, but no 3.0 version yet